ansible_base/rbac/management/__init__.py

```diff
@@ -86,6 +86,11 @@ def create_dab_permissions(app_config, verbosity=2, interactive=True, using=DEFA
             permission.content_type = ct
             perms.append(permission)
 
+    if not perms:
+        return False
+
     Permission.objects.using(using).bulk_create(perms)
     for perm in perms:
         logger.debug("Adding permission '%s'" % perm)
```

Comment:
instead of adding `if not perms:` as a separate block, it might be cleaner to add an `else:` and `return False` there. In terms of code flow...